apps/tx-builder/src/components/Dashboard.tsx

```diff
@@ -108,39 +115,34 @@ const Dashboard = (): ReactElement => {
 
       {/* ABI or Address Input */}
       <StyledAddressInput
-        id={'address-or-ABI-input'}
-        name="addressOrAbi"
-        label="Enter Address, ENS Name or ABI"
+        id="address"
+        name="address"
+        label="Enter Address or ENS Name"
         hiddenLabel={false}
-        address={addressOrAbi}
+        address={address}
         showNetworkPrefix={!!chainInfo?.shortName}
         networkPrefix={chainInfo?.shortName}
-        error={!isValidAddressOrContract ? loadContractError : ''}
+        error={!contractHasMethods ? loadContractError : ''}
         showLoadingSpinner={isABILoading}
         getAddressFromDomain={getAddressFromDomain}
-        onChangeAddress={(addressOrAbi: string) => setAddressOrAbi(addressOrAbi)}
+        onChangeAddress={(address: string) => setAddress(address)}
         InputProps={{
-          endAdornment: isValidAddressOrContract && (
+          endAdornment: contractHasMethods && isValidAddress(address) && (
             <InputAdornment position="end">
               <CheckIconAddressAdornment />
             </InputAdornment>
           ),
         }}
       />
 
-      {/* ABI Warning */}
-      {isValidAddressOrContract && !contract && (
-        <StyledWarningText color="warning" size="lg">
-          No ABI found for this address
-        </StyledWarningText>
-      )}
+      <JsonField id={'abi'} name="abi" label="Enter JSON" value={abi} onChange={setAbi} />
```

Comment:
I think that we can use another label right? because I think that "Enter JSON" is too generic for the user
Something like "Enter ABI" ?

![Captura de pantalla 2022-02-09 a las 10 54 05](https://user-images.githubusercontent.com/26763673/153172033-818ef2de-0024-4c7f-a4c6-ce1961eefa19.png)